source/lang/golang/rabbitgo.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	s "strings"
+)
+
+type JsonData struct {
+	From string `json:"from"`
+	To   string `json:"to"`
+}
+
+func zg2uni(input string) string {
```

Comment:
In Go, if something starts with a capital letter that means other packages (and programs) are able to see it. If you had named the function `zg2uni` instead of `Zg2uni`, your functions or methods would not have been able to see it.